scripts/codes/opss-benefit-type.json

```diff
@@ -0,0 +1,4 @@
+{"_id":"ed0e77d7-a2f6-4b89-8f9b-713494564324","code":"career-average","label":"Career Average","links":{"code_list":{"id":"opss-benefit-type","href":"http://localhost:22400/code-lists/opss-benefit-type"},"self":{"id":"career-average","href":"http://localhost:22400/code-lists/opss-benefit-type/codes/career-average"}}}
```

Comment:
Could the labels be sentence cased please? Like 'All benefit types' is lower case, 'Career average' etc